src/utils/history-sort.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var _ = require('underscore');
+var dateStringified = require('./date-stringify');
+
+module.exports = function(history) {
+  var result = [];
+  
+  _.each(history.components, function(singleComponent, name) {
```

Comment:
Agreed @nickbalestra, especially about the quick sort. That sort and reverse will not make it to the final version 👍